dataprofiler/data_readers/base_data.py

```diff
@@ -72,6 +72,10 @@ def data(self):
     def data_format(self):
         return self._selected_data_format
 
+    @property
+    def is_structured(self):
```

Comment:
let's add a comment describing the property. I know others may not, but we should add those in another task and make sure to add in future.

Reply:
Is this necessary?

Reply:
I believe it should clarify the intent, no?

Reply:
In terms of if it should be in base? Yes, I believe all our readers should clarify hence it should be in the abstract so anyone deriving from the base would know to include it.